src/components/StaticFilters.tsx

```diff
@@ -48,9 +32,9 @@ export interface StaticFiltersProps {
   /** The fieldId corresponding to the static filter group. */
   fieldId: string,
   /** {@inheritDoc FilterOptionConfig} */
-  filterOptions: FilterOptionConfig[],
+  filterOptions: StaticFilterOptionConfig[],
   /** The displayed label for the static filter group. */
-  title?: string,
+  title: string,
```

Comment:
it seems odd that the title is required here when it's optional in FilterGroup. I think we should keep it consistent

Reply:
oh I think it should be required in FilterGroup actually